spec/requests/items_requests_spec.rb

```diff
@@ -32,6 +32,40 @@
       end
     end
 
+    describe "GET #new" do
+      it "shows the organization request_units options if they exist" do
+        Flipper.enable(:enable_packs)
+        organization_units = create_list(:unit, 3, organization: organization)
+        get new_item_path
+        organization_units.each do |unit|
+          expect(response.body).to include unit.name
+        end
+      end
+    end
+
+    describe "GET #edit" do
+      it "shows the selected request_units" do
+        Flipper.enable(:enable_packs)
+        organization_units = create_list(:unit, 3, organization: organization)
+        selected_unit = organization_units.first
+        item = create(:item, organization: organization)
+        create(:item_unit, item: item, name: selected_unit.name)
+
+        get edit_item_path(item)
+
+        parsed_body = Nokogiri::HTML(response.body)
```

Comment:
I wanted to see that we're actually checking the checkbox and this was the only way I could get it to work as Nokogiri doesn't supported the `:checked` pseudo selector that would greatly clean this up

Reply:
Is fine